src/main/java/appeng/me/cache/GridStorageCache.java

```diff
@@ -66,6 +72,32 @@ public GridStorageCache(final IGrid g) {
     public void onUpdateTick() {
         this.itemMonitor.onTick();
         this.fluidMonitor.onTick();
+
+        this.itemBytesTotal = 0;
+        this.itemBytesUsed = 0;
+        try {
```

Comment:
It will update on each tick. I think is not necessary but can take a time to iterate over all of the storages. It would be better if you move it into separate monitor will updated from here but can skip most of update one in 5 secs is good enough resolution for statistic

Reply:
You can have just int field inside and increment it on each tick but calculate all things only when ticks / 1000 == 0 and reset variable into 0

Reply:
I thought this method was not allowed

Reply:
i  done it in the newest push